source/isaaclab/isaaclab/managers/reward_manager.py

```diff
@@ -141,8 +141,11 @@ def compute(self, dt: float) -> torch.Tensor:
         self._reward_buf[:] = 0.0
         # iterate over all the reward terms
         for name, term_cfg in zip(self._term_names, self._term_cfgs):
```

Comment:
Noticed this line of code because of this MR. Why do we index term_names in the loop to get the index of the reward term. Finding an item in a list in such a loop is expensive. It is easily fixable by just doing an enumerate with the zip.

Could this be fixed?

Reply:
@Mayankm96 Thank you for pointing that out. I have fixed that and pushed a change.

Reply:
Thank you for the prmoptness!